src/modules/land_detector/LandDetector.h

```diff
@@ -162,10 +149,17 @@ class LandDetector : public ModuleBase<LandDetector>, ModuleParams, px4::Schedul
 
 	actuator_armed_s         _actuator_armed{};
 	vehicle_acceleration_s   _vehicle_acceleration{};
-	vehicle_land_detected_s  _land_detected{};
+	vehicle_land_detected_s _land_detected = {
+		.timestamp = 0,
+		.alt_max = -1.0f,
+		.freefall = false,
+		.ground_contact = true,
+		.maybe_landed = true,
+		.landed = true,
+	};
```

Comment:
I think this was intentional and it is correct to initialize to the new values you have, but just in case it was not intended I wanted to point out the master branch current constructor initialized values:
```
	_land_detected.ground_contact = false;
	_land_detected.maybe_landed = false;
```

Reply:
We could leave this initializer like it was, the only place it get published is in the `Run()` method and there it also overwrites all of the struct content.

Reply:
I think it's good the way you have it.  Just wanted to point it out in case. :)